airflow/dags/gtfs_views_staging/validation_notices_clean.sql

```diff
@@ -5,7 +5,55 @@ dependencies:
   - type2_loaded
 ---
 
+-- Must trim string fields that come from raw GTFS tables that we clean & load into views
+-- (To allow joining with the cleaned data after this is run)
+-- Don't trim stopId because we don't load stops into views
+-- This table has over 70 columns, so even though EXCEPT is a bit messy it still seems cleaner
+
 SELECT
-    * EXCEPT(calitp_deleted_at)
+    * EXCEPT(
```

Comment:
I don't believe we do any joins of the validation notices data back to any of the daily fact tables, but I guess this doesn't hurt.